packages/platform/atoms/hooks/useCalendarsBusyTimes.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from "@tanstack/react-query";
+
+import type { ApiResponse, CalendarBusyTimesInput } from "@calcom/platform-types";
+import type { EventBusyDate } from "@calcom/types/Calendar";
+
+import http from "../lib/http";
+
+export const QUERY_KEY = "get-calendars-busy-times";
+
+export const useCalendarsBusyTimes = (props: CalendarBusyTimesInput) => {
+  const availableSlots = useQuery({
+    queryKey: [QUERY_KEY],
+    queryFn: () => {
+      return http
+        .get<ApiResponse<EventBusyDate[]>>("/ee/calendars/busy-times", {
```

Comment:
Similar to get connected calendars [PR](https://github.com/calcom/cal.com/pull/13757/files)  "/ee/calendars/busy-times" can be added to `const V2_ENDPOINTS`